job_spec/INSAR_MULTI_BURST.yml

```diff
@@ -0,0 +1,88 @@
+INSAR_MULTI_BURST:
+  required_parameters:
+    - reference
+    - secondary
+  parameters:
+    reference:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 1
+        maxItems: 50
+        example:
+          - S1_136231_IW2_20200604T022312_VV_7C85-BURST
+          - S1_136232_IW2_20200604T022312_VV_7C85-BURST
+        items:
+          description: Name of the reference Sentinel-1 SLC IW burst granule to process
+          type: string
+          pattern: '^((S1_\d{6}_IW\d_\d{8}T\d{6}_[VH]{2}_([\dA-F]){4}-BURST)($|[ \t]+))+$'
+          minLength: 43
+          example: 
+            - S1_136231_IW2_20200604T022312_VV_7C85-BURST
+    secondary:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 1
+        maxItems: 50
+        example:
+          - S1_136231_IW2_20200616T022313_VV_5D11-BURST
+          - S1_136232_IW2_20200616T022313_VV_5D11-BURST
+        items:
+          description: Name of the secondary Sentinel-1 SLC IW burst granule to process
+          type: string
+          pattern: '^((S1_\d{6}_IW\d_\d{8}T\d{6}_[VH]{2}_([\dA-F]){4}-BURST)($|[ \t]+))+$'
+          minLength: 43
+          example: S1_136231_IW2_20200616T022313_VV_5D11-BURST
+    bucket_prefix:
+      default:  '""'
+    apply_water_mask:
+      api_schema:
+        description: Sets pixels over coastal and large inland waterbodies as invalid for phase unwrapping. 
+        default: false
+        type: boolean
+    looks:
+      api_schema:
+        description: Number of looks to take in range and azimuth
+        type: string
+        default: 20x4
+        enum:
+          - 20x4
+          - 10x2
+          - 5x1
+  cost_profiles:
+    EDC:
+      cost: 1.0
+    DEFAULT:
+      cost: 1.0
```

Comment:
We have an open Jira item for changing the cost, right?

Reply:
Yes. This is what the single burst workflow uses.